packages/fs.go

```diff
@@ -35,23 +36,29 @@ func NewExtractedPackageFileSystem(p *Package) (*ExtractedPackageFileSystem, err
 }
 
 func (fs *ExtractedPackageFileSystem) Stat(name string) (os.FileInfo, error) {
-	return os.Stat(filepath.Join(fs.path, name))
+	return os.Stat(path.Join(fs.path, name))
```

Comment:
I think that when accessing the file system directly, we should keep using `filepath`, that takes into account the different separator in Windows. This would be for all methods of `ExtractedPackageFileSystem`, that work on the native file system.

As a rule of thumb, if the path is used with a function of `os` package, it should be manipulated with `filepath` package.

Reply:
If we want to work with `path`, we could use [`os.DirFS`](https://pkg.go.dev/os#DirFS) to build a `fs.FS`.